source/js/admissions.js

```diff
@@ -0,0 +1,36 @@
+//import { ModalHandler, ModalViewer, ModalContainer } from './modules/Modal.js';
+
+const modalBox = document.getElementById("modal-box");
+const modalCloseBtn = document.getElementById("modal-close");
+const modalBckg = document.getElementById("modal-background");
+
+var Handler;
+var Viewer = new ModalViewer(modalBox, modalBckg, modalCloseBtn);
+var Container;
+
+
+$(modalCloseBtn).click((e) => {
+    if (!Handler) {
+        e.preventDefault();
+        throw new Error("Handler was not set before calling modalCloseBtn clickEvent");
+    }
+    Handler.hide();
+});
+
+// var btn = Viewer._addButton("modal-inner-close", "Back"); - Fix event listener getting ducked up when shown
```

Comment:
why not just remove?

Reply:
I'll leave it there for now, it's a reminder for me to have a look at it and fix it. I may remove it later tho.